lib/ocpp/common/message_queue.cpp

```diff
@@ -7,43 +7,59 @@
 
 namespace ocpp {
 
-template <> ControlMessage<v16::MessageType>::ControlMessage(const json& message) {
-    this->message = message.get<json::array_t>();
-    this->messageType = v16::conversions::string_to_messagetype(message.at(CALL_ACTION));
-    this->message_attempts = 0;
-    this->initial_unique_id = this->message[MESSAGE_ID];
-}
+MessageTransmissionPriority get_message_transmission_priority(bool is_boot_notification_message, bool triggered,
```

Comment:
Just a heads up, this file does not exist anymore since the v16/v201 compilation change so you'd have to split the changes up too.